Dangerfile

```diff
@@ -192,6 +204,48 @@ module VSPDanger
     end
   end
 
+  class GemfileProtector
+    def run
+      return error if bad_gemfile_changes?
+
+      gemfile_ok_message
+    end
+
+    private
+
+    def error
+      { severity: :error, message: error_message }
+    end
+
+    def bad_gemfile_changes?
+      gemfile_diff.include?('-  remote: https://enterprise.contribsys.com/')
+    end
+
+    def error_message
+      <<~EMSG
+        You've removed Sidekiq Enterprise from the gemfile!  You must restore it before merging this PR.
+
+        More details about Sidekiq Enterprise can be found in the [README](https://github.com/department-of-veterans-affairs/vets-api/blob/master/README.md).
+      EMSG
+    end
+
+    def gemfile_ok_message
+      { severity: :info, message: 'Gemfile changes acceptable.' }
+    end
+
+    def gemfile_diff
+      `git diff #{base_sha}:Gemfile.lock #{head_sha}:Gemfile.lock`
+    end
+
+    def head_sha
+      `git rev-parse --abbrev-ref HEAD`.chomp
+    end
+
+    def base_sha
+      'origin/master'
+    end
```

Comment:
We're reusing these a lot (head_sha and base_sha). Could we extract them somewhere? Maybe module constants?

Reply:
I was wondering about that too.  That sounds like a good place for them!